mirdata/datasets/jtd.py

```diff
@@ -0,0 +1,722 @@
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    @property
+    def audio_lchan(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The track's audio, left channel (not present for all tracks)
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_lchan_path)
+
+    @property
+    def audio_rchan(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The track's audio, right channel (not present for all tracks)
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_rchan_path)
+
+    @property
+    def bandleader(self) -> Optional[str]:
+        """The full name of the bandleader who led the recording session
+
+        Returns:
+            * str - name of the bandleader
+
+        """
+        return (
+            self._multitrack_metadata["bandleader"]
+            if "bandleader" in self._multitrack_metadata
+            else None
+        )
+
+    @property
+    def bass(self) -> Track:
+        """The associated bass track for this recording
+
+        Returns:
+            * Track
+
+        """
+        return self.tracks[self.mtrack_id + "_bass"]
+
+    @core.cached_property
+    def beats(self) -> Optional[annotations.BeatData]:
+        """The times of quarter-note beats for the recording
+
+        Returns:
+            * annotations.BeatData - timestamp, beat number (1-indexed to bar)
+
+        """
+        return load_beats(self.beats_path, 0)
+
+    @property
+    def drums(self) -> Track:
+        """The associated drums track for this recording
+
+        Returns:
+            * Track
+
+        """
+        return self.tracks[self.mtrack_id + "_drums"]
+
+    @property
+    def duration(self) -> Optional[int]:
+        """The duration of the piano solo
+
+        Returns:
+            * float - solo duration (in seconds)
+
+        """
+        start = self._multitrack_metadata["timestamps"]["start"]
+        stop = self._multitrack_metadata["timestamps"]["end"]
+        return timestamp_to_seconds(stop) - timestamp_to_seconds(start)
+
+    @property
+    def jtd_300(self) -> Optional[bool]:
+        """Whether the track is contained in the smaller JTD-300 subset of 300 recordings
+
+        Returns:
+            * bool - True if contained in JTD-300, otherwise false
+
+        """
+        return (
+            self._multitrack_metadata["in_30_corpus"]
+            if "in_30_corpus" in self._multitrack_metadata
+            else None
+        )
+
+    @property
+    def musicbrainz_id(self) -> Optional[str]:
+        """The MusicBrainz ID for the recording
+
+        Returns:
+            * str - musicbrainz ID
+
+        """
+        return (
+            self._multitrack_metadata["mbz_id"]
```

Comment:
Consider simplifying these kinds of expressions to 

```py
self._multitrack_metadata.get("mbz_id", None)
```

Reply:
Thanks, I'd agree that your suggestion is more pythonic @pmcharrison but I've seen this syntax used in other loaders within `mirdata` e.g. `cipi.py`. 

Will wait for more opinions!